cloud_formation/configs/activities.py

```diff
@@ -138,14 +207,18 @@ def post_init(session, domain):
     sfn.create(session, names.ingest_queue_populate, domain, 'ingest_queue_populate.hsd', 'StatesExecutionRole-us-east-1 ')
     sfn.create(session, names.ingest_queue_upload, domain, 'ingest_queue_upload.hsd', 'StatesExecutionRole-us-east-1 ')
     sfn.create(session, names.resolution_hierarchy, domain, 'resolution_hierarchy.hsd', 'StatesExecutionRole-us-east-1')
-    sfn.create(session, names.downsample_volume, domain, 'downsample_volume.hsd', 'StatesExecutionRole-us-east-1')
+    #sfn.create(session, names.downsample_volume, domain, 'downsample_volume.hsd', 'StatesExecutionRole-us-east-1')
 
 
 def delete(session, domain):
-    names = AWSNames(domain)
     # DP TODO: delete activities
```

Comment:
Is this still a todo?

Reply:
yes. not critical. basically each Step Function Activity has an ARN that should be deleted when the Step Function is deleted. If you look under the Step Function console there is a list of Activities and old / deleted Activities are still listed.